### ISSUE TITLE
SCREEN_share Hotkey

### ISSUE THREAD

**Issue by Blazzycrafter** [Vencord/plugin-requests#712]
### explenation
i used pageUp to start an screenshare...
but everytime its only when i play a game...
but if i want share my screen as example writing text or yt, or chatgpt... then i must go ito the app to start it....

### solution
so plase make an plugin that simply replaces the screenshare(hotkey) start code to start the stream in two ways:

1. static: screenshare screen 1 or screen 2 (based on settings)
2. dynamic: screenshare the screen where:
- the mouse is
- where the active window is

### Settings ideas
sry for the `pseudo code`
- dynamic: BOOL
- screen selection: INT (enabled: `if Settings.dynamic == FALSE: return TRUE ;else: return FALSE`)

### Request Agreement

- [x] I read the above and made sure my plugin is possible and allowed
- [X] I checked the [existing plugins](https://vencord.dev/plugins) and made sure my plugin doesn't already exist
- [X] I [searched the existing issues](https://github.com/Vencord/plugin-requests/issues?q=is%3Aissue) and made sure a similar request doesn't already exist

**Comment by Fummowo** [comment]
I only ever stream the OBS VCam so this would be incredibly handy if hotkeys could be setup for streaming Capture Devices aswell and +1 for picking the Audio Device for that too :))

**Comment by dBerkeTurkmen** [comment]
I definitely need such a plugin. Please someone do it for us!

**Comment by Sqaaakoi** [comment]
isn't this a thing in stock Discord, but it only streams the **current game**?

**Comment by dBerkeTurkmen** [comment]
@Sqaaakoi Yes, the Discord feature only shares the game that is on the screen.
